### PR TITLE
Fixes Active Membership Requirements and Packet

### PR DESCRIPTION
**Pull request by saucetray**
Active Membership is now awarded upon intro-members if they pass 10-weeks.

Check one:
- [x] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -186,7 +186,7 @@
 
 \asection{Active Membership}
 \asubsection{Active Membership Qualifications}
-Active Membership is open to all students currently enrolled at the Rochester Institute of Technology who have passed the Introductory Evaluation and their most recent Membership Evaluation.
+Active Membership is open to all students currently enrolled at the Rochester Institute of Technology who have passed their last Introductory Evaluation or their most recent Membership Evaluation.
```
Comment (mxmeinhold) [inline constitution.tex]:
```suggestion
Active Membership is open to all students currently enrolled at the Rochester Institute of Technology who have passed their most recent Introductory Evaluation or their most recent Membership Evaluation.
```
We should be consistent.

File: constitution.tex [hunk 2]
```diff
@@ -522,7 +523,7 @@
 The Process will begin between the first and fourth week of each semester, then last between six (6) and ten (10) weeks, starting when the Evaluations Director initiates it.
 The Executive Board may hold a closed ballot Simple Majority vote to extend the Introductory Process or the Introductory Packet.
 \asubsubsubsection{The Introductory Packet}
-Each Introductory Process participant, after the first week, is given two (2) weeks to return a document that should contain a signature from all Active members, all Introductory Resident members, and ten (10) of any combination of the following:
+Each Introductory Process participant, after the first week, is given two (2) weeks to return a document that should contain a signature from all Active members whom passed a Membership Evaluation, all Introductory Resident members, and ten (10) of any combination of the following:
```
Comment (mxmeinhold) [inline constitution.tex]:
```suggestion
Each Introductory Process participant, after the first week, is given two (2) weeks to return a document that should contain a signature from all Active members whom have passed a Membership Evaluation, all Introductory Resident members, and ten (10) of any combination of the following:
```

File: constitution.tex [hunk 3]
```diff
@@ -186,7 +186,7 @@
 
 \asection{Active Membership}
 \asubsection{Active Membership Qualifications}
-Active Membership is open to all students currently enrolled at the Rochester Institute of Technology who have passed the Introductory Evaluation and their most recent Membership Evaluation.
+Active Membership is open to all students currently enrolled at the Rochester Institute of Technology who have passed their most recent Introductory Evaluation or their most recent Membership Evaluation.
```
Comment (mxmeinhold) [inline constitution.tex]: I don't see what changing this to `or` fixes. I think it opens more loopholes.

File: constitution.tex [hunk 4]
```diff
@@ -522,7 +523,7 @@
 The Process will begin between the first and fourth week of each semester, then last between six (6) and ten (10) weeks, starting when the Evaluations Director initiates it.
 The Executive Board may hold a closed ballot Simple Majority vote to extend the Introductory Process or the Introductory Packet.
 \asubsubsubsection{The Introductory Packet}
-Each Introductory Process participant, after the first week, is given two (2) weeks to return a document that should contain a signature from all Active members, all Introductory Resident members, and ten (10) of any combination of the following:
+Each Introductory Process participant, after the first week, is given two (2) weeks to return a document that should contain a signature from all Active members whom have passed a Membership Evaluation, all Introductory Resident members, and ten (10) of any combination of the following:
```
Comment (MasterChief-John-117) [inline constitution.tex]: Grammatically, whom should be who

**Review by devinmatte — COMMENTED** [review]
Is this looking to still be brought up?

**Pull request closed without merging**